db.go

```diff
@@ -75,12 +75,26 @@ func (self *DB) Query(sql string, vars map[string]any) (any, error) {
 	return self.send("query", sql, vars)
 }
 
+func (self *DB) SchemalessSelect(what string) ([]map[string]interface{}, error) {
+	output, err := self.Select(what)
+	if err != nil {
+		return make([]map[string]interface{}, 0), err
+	}
+
+	returnArray := make([]map[string]interface{}, 0)
+	rows := (output).([]interface{})
```

Comment:
This will crash if output is not a `[]interface{}` (we probably don't want that)

Reply:
Fixed -- made the code flow better anyways -- so thanks for pointing this out